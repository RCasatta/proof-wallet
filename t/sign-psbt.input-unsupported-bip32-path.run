#!/bin/bash

../../glacierscript.py sign-psbt -m 2 -n 2 --regtest << INPUT 2>&1
y
y
y
y
y
y
mouse scout chest give trim veteran keen arrive test advice warrior reduce silent useful oak embrace drift curve large poet embody theme focus wolf
tpubD6NzVbkrYhZ4YTjZqdGdMpqtd7BFKkERJcK3hjQx4gtHC3xowC5EKAsHY5tgx1coSC6sbw6X7Ngc1AFrVEzgwFFAF8pbsU5n28yYiZrxTAg
tpubD6NzVbkrYhZ4XcWFWZiEr9DBG7eFbG5H6Lz4TwmMw9FwhaRT7Yj5LyPFFBp7jok8uaPKWYsQtHEK2e1N231CmVt2QfMsjCn46by1NbrFEq9
cHNidP8BAIACAAAAAURrW7a/0IjQJW4qN6xld6NLFTSoGe+WA2VzxApER5svAAAAAAD9////AnzPAwAAAAAAIgAg0oDDltDjDejjGuz/2oGhg0ErfFOuzUJ7fnjQKNgLVGqwcQsAAAAAABl2qRQ0Sg9IyhUOwrkDgXZgubaLE6ZwJoisWXcZAAABAStAQg8AAAAAACIAIObsDL3M58bQU3k5T9aWSPQGPn+pZ4Xm41qatgr3rL1IAQVHUiECAi9g1wJg9FyhlZTAdxINPYA106IBvhSdmmvq3d4D954hA/RVGRXGGIAvve4NCiT2R/vc3fuXYuGTCmyAnsmNyjSAUq4iBgICL2DXAmD0XKGVlMB3Eg09gDXTogG+FJ2aa+rd3gP3nhA06TzaAAAAAAAAAAAAAAAAIgYD9FUZFcYYgC+97g0KJPZH+9zd+5di4ZMKbICeyY3KNIAQC4ipNwAAAAAAAAAAAAAAAAABAUdSIQJIpi3B16WvnfaxAok7Bj3y3FHgNkdyYmD8fZW9y/evGSECpjgib8VI58A25EhnGrmHPFDlZQcTMaYB/siGkCnQmtRSriICAkimLcHXpa+d9rECiTsGPfLcUeA2R3JiYPx9lb3L968ZDAuIqTcBAAAAAAAAACICAqY4Im/FSOfANuRIZxq5hzxQ5WUHEzGmAf7IhpAp0JrUDDTpPNoBAAAAAAAAAAAA
INPUT

retVal=$?
if [ $retVal -eq 0 ]; then
    echo "Error: expected glacierscript.py to fail"
    exit 1
fi
