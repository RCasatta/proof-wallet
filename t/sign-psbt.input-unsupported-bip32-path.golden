Are you running this on a computer WITHOUT a network connection of any kind? (y/n)?Have the wireless cards in this computer been physically removed? (y/n)?Are you running on battery power? (y/n)?Are you running on an operating system booted from a USB drive? (y/n)?Is your screen hidden from view of windows, cameras, and other people? (y/n)?Are smartphones and all other nearby devices turned off and in a Faraday bag? (y/n)?
Enter your BIP39 mnemonic phrase (separate the words with whitespace): 
Input 3 valid descriptor keys

Enter descriptor key #1: 
Enter descriptor key #2: 
Enter descriptor key #3: 
Enter the psbt for the transaction you wish to sign: 
Validating the PSBT...
Error: Tx input 0 contains an unsupported derivation path: '../0/0/2'.
